examples/step-62/doc/intro.dox

```diff
@@ -0,0 +1,220 @@
+simulations. A pulse at a certain frequency is generated on the left side of the
+structure and the transmitted energy is measured on the right side of the structure.
+The simulation is run twice. First, we run the simulation with the phononic
+structure and measure the transmitted energy.
+
+<img alt="Phononic superlattice cavity" src="https://raw.githubusercontent.com/dangars/dealii/phononic-cavity/examples/step-62/doc/step-62.02.svg?sanitize=true" height="200" />
+
+Then we run the simulation without the phononic structure and measure the transmitted
+energy; we use the simulation without the structure for the calibration.
+
+<img alt="Phononic superlattice cavity" src="https://raw.githubusercontent.com/dangars/dealii/phononic-cavity/examples/step-62/doc/step-62.03.svg?sanitize=true" height="200" />
+
+The transmission coefficient corresponds to the energy of the first simulation
+divided by the calibration energy.
+We repeat this procedure for each frequency step.
+
+
+<h3>Elastic equations</h3>
+The elastic equations in the time domain are 
+@f[
+\rho\partial_{tt} u_i - \partial_j (c_{ijkl} \varepsilon_{kl}) = f_i,
+\qquad i=0,1,2
+@f]
+where the stiffness tensor $c_{ijkl}$ depends on the spacial coordinates and
+the strain is given by
+@f[
+\varepsilon_{kl} =\frac{1}{2}(\partial_k u_l + \partial_l u_k)
+@f]
+
+[A perfectly matched layer (PML)](https://en.wikipedia.org/wiki/Perfectly_matched_layer)
+can be used to truncate the solution at the boundaries.
+A PML is a transformation that results in a complex coordinate stretching.
+The elastic equations in the frequency domain read as follows
+@f{eqnarray*}
+\nabla\cdot(\boldsymbol{\bar\sigma} \xi \boldsymbol{\Lambda})&=&-\omega^2\rho\xi\mathbf{\bar u}\\
+\boldsymbol{\bar \sigma} &=&\mathbf{C}\boldsymbol{\bar\varepsilon}\\
+\boldsymbol{\bar\varepsilon}&=&\frac{1}{2}[(\nabla\mathbf{\bar{u}}\boldsymbol{\Lambda}+\boldsymbol{\Lambda}^\mathrm{T}(\nabla\mathbf{\bar{u}})^\mathrm{T})]\\
+\xi &=&s_0\cdot s_1\cdot s_2\\
+\boldsymbol{\Lambda} &=& \operatorname{diag}(1/s_0,1/s_1,1/s_2)
+@f}
+where the coefficients $s_i = 1+is_i'(x,y,z)$ account for the absorption.
+The imaginary par of $s_i$ is equal to zero outside of the PML.
+The PMLs are reflectionless only for the exact wave equations.
+When the set of equations is discretized the PML is no longer reflectionless.
+The reflections can be made arbitrarily small as long as the
+medium is slowly varying, see
+[the adiabatic theorem](https://doi.org/10.1103/PhysRevE.66.066608).
+In the code a quadratic turn-on of the PML has been used.
+A linear and cubic turn-on is also
+[known to work](https://doi.org/10.1364/OE.16.011376).
+These equations can be expanded into
+@f[
+-\omega^2\rho \xi  u_m - \partial_n \left(\frac{\xi}{s_n}c_{mnkl}
+\varepsilon_{kl}\right) = f_m
+@f]
+@f[
+\varepsilon_{kl} =\frac{1}{2}\left(\frac{1}{s_k}\partial_k u_l
++ \frac{1}{s_l}\partial_l u_k\right)
+@f]
```

Comment:
The strain with the PML is not symmetric because of the `s_k` coefficients. The strain with the PML is slightly different that the strain from the standard elastic equation. Now I explain that in the text.

Reply:
I see. I missed that, so others might as well. Maybe say something like this explicitly, rather than just showing a formula?